### PR TITLE
Design 0204

### PR DESCRIPTION
**Pull request by manibi**
@CristianCristea pls review, merge & push

### CODE REVIEW

File: app/views/users/_generated_temp_user.html.erb [hunk 1]
```diff
@@ -12,27 +12,115 @@
   <% if user.feedback %>
```
Comment (CristianCristea) [inline app/views/users/_generated_temp_user.html.erb:12]: Details btn only appears when the user leaves a review. We want the details all the time. Remove if user.feedback?

File: app/views/users/_generated_temp_user.html.erb [hunk 2]
```diff
@@ -12,27 +12,115 @@
...
+      <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#newTempUserFeedback<%= user.id %>">
+        Details
       </button>
     </td>
   <% end %>
 </tr>
 
+<%# raise %>
 
 <% if user.feedback %>
```
Comment (CristianCristea) [inline app/views/users/_generated_temp_user.html.erb:24]: Remove if user.feedback?

Comment (manibi) [reply app/views/users/_generated_temp_user.html.erb]: Done!

File: app/views/users/_generated_temp_user.html.erb (hunk 1 above)
Comment (manibi) [reply app/views/users/_generated_temp_user.html.erb]: -done-

**Pull request merged**